tests/test_suite-docker.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python
```

Comment:
The test suite does not have any test output. Please add output.
```
[jcerny@thinkpad tests{ssgts}]$ python3 ./test_suite-docker.py profile --base-image ssg_test_suite --datastream ../build/ssg-centos7-ds.xml --xccdf-id scap_org.open-scap_cref_ssg-rhel7-xccdf-1.2.xml xccdf_org.ss
gproject.content_profile_stig-rhel7-disa
Setting console output to log level INFO
INFO - Logging into /home/jcerny/scap-security-guide/tests/logs/profile-custom-2018-08-17-1048/test_suite.log                                                                                                     
INFO - Evaluation of profile xccdf_org.ssgproject.content_profile_stig-rhel7-disa.
```

Reply:
The output of the test suite is the same as it used to be. I understand that you don't like it so much, but it is out of scope of this PR.